src/Behat/Behat/Context/Argument/NullFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Context\Argument;
+
+use Behat\Testwork\Suite\Suite;
+
+/**
+ * NoOp factory. Always returns zero resolvers.
+ *
+ * @see ContextEnvironmentHandler
+ *
+ * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ */
+final class NullFactory implements SuiteScopedResolverFactory
```

Comment:
There might be better naming - `noop` from the comment makes sense, or maybe `empty`?